resolwe/process/models.py

```diff
@@ -261,10 +265,12 @@ def default(self, o: Any):
 class Model(metaclass=ModelMetaclass):
     """Base django model."""
 
-    def __init__(self, model_name: str, pk: int):
+    _app_name = "App name"
```

Comment:
It would be nice to have a sanity check that these values are defined in subclasses.